README.md

```diff
@@ -22,410 +27,10 @@ sasl.username=admin
 sasl.password=<REDACTED>
 ```
 
-## Overview
```

Comment:
This was moved to [provisioning/broker/README.md](./provisioning/broker/README.md).